ckks/encoder.go

```diff
@@ -232,11 +233,18 @@ func (encoder *encoderComplex128) GetErrSTDSlotDomain(valuesWant, valuesHave []c
 // GetErrSTDCoeffDomain returns the scaled standard deviation in the coefficient domain of the difference between two complex vectors in the slot domains
 func (encoder *encoderComplex128) GetErrSTDCoeffDomain(valuesWant, valuesHave []complex128, scale float64) (std float64) {
 
+	encoder.WipeInternalMemory()
+
+	nextPow2 := 1
+	for nextPow2 < len(valuesWant) {
+		nextPow2 <<= 1
+	}
```

Comment:
Can this loop be replaced by `1 << bits.Len64(uint64(x-1))` ? 

I agree that a loop-free version is less explicit but you can  compensate with a code comment.

Reply:
It can, but then the file would need to import math/bits just for that line.

Reply:
I don't see any problem in importing `math/bits`. What would it be ?

Reply:
Made the required changes